csharp/ql/src/semmle/code/csharp/controlflow/internal/Splitting.qll

```diff
@@ -525,53 +551,90 @@ module FinallySplitting {
       (exists(succ(pred, _)) or exists(succExit(pred, _)))
     }
 
-    /** Holds if `pred` may exit this split with completion `c`. */
-    private predicate exit(ControlFlowElement pred, Completion c) {
+    /**
+     * Holds if `pred` may exit this split with completion `c`. The Boolean
+     * `derived` indicates whether `c` is a derived completion from a `try`/
```

Comment:
Jargon "derived" is not defined - a little note would be helpful?

Reply:
OK, we have been using "inherited" elsewhere for this, so I will switch to that.